srcpkgs/artyfx/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'artyfx'
+pkgname=artyfx
+version=1.3
+revision=1
+wrksrc=openAV-ArtyFX-release-$version
+build_style=cmake
+makedepends="cairo-devel libsndfile-devel lv2 pkg-config"
+depends="cairo libsndfile"
```

Comment:
not needed

Reply:
Which bit isn't needed?

Reply:
the line he specifically commented on.

Why: Because those libraries are found automatically by scanning all ELF binaries.

Reply:
That's... really clever. XBPS is awesome.